development/scripts/add-rn-startup-performance-tracking.js

```
@@ -0,0 +1,189 @@
+}
+
+export {};
+`;
+
+    // Write ready file
+    fs.writeFileSync(readyFilePath, readyFileContent, 'utf-8');
+    stats.readyFilesCreated++;
+
+    // Modify original file
+    const importStatement = `import './${fileNameWithoutExt}.perfReady';\n`;
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)